.ci/pytorch/smoke_test/smoke_test.py

```diff
@@ -451,6 +502,7 @@ def main() -> None:
     smoke_test_conv2d()
     test_linalg()
     test_numpy()
+    test_gomp()
```

Comment:
Hi @LifengWang  Can we add such test into `.ci/pytorch/check_binary.sh` directly?